src/main/webapp/features/screens/game/view.html

```diff
@@ -1,48 +1,3 @@
-<div class="row">
+<ct-game-room ng-if="game.isNew()" game="game"></ct-game-room>
```

Comment:
I believe we can. Do you remember how tournaments happen in poker rooms? at the beginning you can see tournament details, and after game is started, you are redirected to your table, but you can still look at tournament details if you wish.